.bazelci/presubmit.yml

```diff
@@ -12,21 +12,37 @@ tasks:
     shell_commands:
       - tests/core/cgo/generate_imported_dylib.sh
     build_targets:
-    - "//..."
+      - "//..."
     test_targets:
-    - "//..."
+      - "//..."
   ubuntu2004:
     # enable some unflipped incompatible flags on this platform to ensure we don't regress.
     shell_commands:
       - tests/core/cgo/generate_imported_dylib.sh
     build_flags:
-    - "--config=incompatible"
+      - "--config=incompatible"
+    test_flags:
+      - "--config=incompatible"
+    build_targets:
+      - "//..."
+    test_targets:
+      - "//..."
+  ubuntu2004_uber_hermetic_cc:
+    platform: ubuntu2004
```

Comment:
not sure where to get the list of open platforms, but is there one that doesn't have a C/C++ compiler installed?

Reply:
This is the list: https://github.com/bazelbuild/continuous-integration/blob/8c5e4b1322ea5751842bfd03d2e03d31dbe1f8f2/buildkite/bazelci.py#L465-L615

I am planning to add macOS cross-compiling into Linux, which is a common scenario